styles/chess.com/catppuccin.user.css

```diff
@@ -84,39 +84,40 @@
     @wq: #piece("w/wq") [];
     @wr: #piece("w/wr") [];
 
-    --theme-background-color: @base !important;
-    --globalBackground: @mantle !important;
-    --globalBackgroundOpaque: @mantle !important;
-    --globalSecondaryBackground: @crust !important;
-    --globalTertiaryBackground: @surface0 !important;
-    --globalAccentBackground: @accent-color !important;
-    --globalSecondaryAccentBackground: darken(@accent-color, 20%) !important;
-    --globalSiteBackground: @base !important;
-    --subtleButtonBackground: hsla(0, 0%, 100%, 0.16);
-    --globalBorder: @surface0 !important;
-    --globalGray: hsla(0, 0%, 100%, 0.08);
-    --globalGraySoft: hsla(0, 0%, 100%, 0.08);
-    --globalColorThemeFull: @text !important;
-    --globalColorThemeHigh: @text;
-    --globalColorThemeMid: @overlay2 !important;
-    --globalColorThemeLow: @overlay0 !important;
-    --globalColorThemeLower: @base !important;
-    --globalColorThemeLink: @sapphire !important;
-    --globalColorThemeBlueToWhite: #fff;
-    --globalColorThemeBlueToMid: hsla(0, 0%, 100%, 0.72);
-    --globalColorThemeBlueToMidHover: hsla(0, 0%, 100%, 0.85);
-    --globalColorThemeBlueToHigh: hsla(0, 0%, 100%, 0.85);
-    --globalColorThemeBlueToHighHover: #fff;
-    --globalColorThemeHighToMid: hsla(0, 0%, 100%, 0.72);
-    --globalColorThemeFullToMid: hsla(0, 0%, 100%, 0.72);
-    --globalColorNeutral50: rgba(0, 0, 0, 0.1);
-    --globalColorNeutral100: rgba(0, 0, 0, 0.2);
-    --globalColorNeutral200: rgba(0, 0, 0, 0.4);
-    --globalOverlayBackground: @base !important;
-    --globalColorWin: @green !important;
-    --globalColorDraw: @accent-color !important;
-    --globalColorLoss: @red !important;
-
+    body {
```

Comment:
I think the fix is to change the dark mode theme selector above to `.dark-mode body`.

Reply:
i think thats better yeah

Reply:
Remove this block now too please (remove the `body {` and the ending bracket below).

Reply:
alr